source/reference/failure-recovery.txt

```diff
@@ -0,0 +1,97 @@
+.. _c2c-failure-recovery:
+
+==========================
+Recovery From Sync Failure
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+:program:`mongosync` temporarily alters some collection characteristics
+during synchronization. The original values are restored during the
+:ref:`commit process <c2c-api-commit>`. However, a network failure or
```

Comment:
I wouldn't mention network failures. This makes it sound like mongosync is not resilient to network failures. I think the only case where customers need to do manual intervention is that there is a permanent failure on the source / mongosync and that they want to switch their application to the destination cluster.